pennylane/interfaces/jax.py

```diff
@@ -566,10 +551,10 @@ def _compute_jvps(jacs, tangents, multi_measurements):
     return jvps
 
 
-def _copy_tape(t, a):
+def _copy_tape(t, a, unwrap=True):
```

Comment:
Can we rename this helper method to `set_parameters_on_copy_and_unwrap`? And perhaps move it to the top of the file, as code above relies on it. Yes, this helper copies the tape, but it entire point is to set parameters and unwrap to numpy.

Reply:
sure. I've certainly augmented its purpose in this PR, a re-naming would fit too

Reply:
This comment repeats bellow.